flexibleengine/provider.go

```diff
@@ -389,6 +387,8 @@ func Provider() *schema.Provider {
 			// importing existing data source
 			"flexibleengine_rds_flavors_v3": rds.DataSourceRdsFlavor(), // v1.34.0
 
+			"flexibleengine_sdrs_domain_v1": sdrs.DataSourceSDRSDomain(),
+
```

Comment:
This name is not expected to be here. It should not under the  `// importing existing data source`